requirements.txt

```diff
@@ -12,6 +12,6 @@ schema==0.7.2
 six==1.15.0
 total-ordering==0.1.0
 tqdm==4.50.2
-tubeup==0.0.19
+tubeup==0.0.20
 urllib3==1.25.11
-youtube-dl==2020.9.20
+youtube-dlc==2020.9.10.26
```

Comment:
```suggestion
youtube-dlc==2020.10.26
```

Reply:
yeah sorry was early in the morning when I did that.